src/api/app/components/build_results_monitor_component.rb

```diff
@@ -41,7 +47,7 @@ def filtered_architecture_names
   end
 
   def status_names
-    raw_data.pluck(:status).uniq
+    Buildresult.avail_status_values
```

Comment:
What is the reason for this change?

Reply:
This change has ist own commit. You can go through all the commits' list to know about why every part of the code was changed.

In this case, this is the title of the commit: "Show all possible status values instead of those returned by build results". Please tell me if there is still something missing, or clarification needed.

Reply:
Sounds good